org.lflang/src/org/lflang/util/LFCommand.java

```diff
@@ -260,8 +264,9 @@ public static LFCommand get(final String cmd, final List<String> args) {
      * @param dir  The directory in which the command should be executed
      * @return Returns an LFCommand if the given command could be found or null otherwise.
      */
-    public static LFCommand get(final String cmd, final List<String> args, final Path dir) {
+    public static LFCommand get(final String cmd, final List<String> args, Path dir) {
         assert cmd != null && args != null && dir != null;
+        dir = toProperDirectory(dir);
```

Comment:
I am not sure if I like this change. I definitely agree that a non-existent directory is a corner case that should be handled somehow, and I overlooked this. However, I find it a bit strange to have this side effect of creating a directory on a function that actually is only supposed to invoke a command. I think the caller should be responsible for ensuring that the directory exists. If it does not, then this is an error, which should probably result in an exception.

Reply:
Yes I agree that if the parameter doesn't exist, it should be considered invalid. However I'm not sure how the caller can guard against this any other way than just writing `Files.createDirectories(dir);` before the call... But, maybe it's clearer anyway if it's written explicitly in the caller, and it's then something desirable